app/src/main/res/layout/favorite_item.xml

```diff
@@ -27,19 +29,18 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:text="Casa"
+                android:textSize="16sp"
                 android:textColor="@color/abc_search_url_text_selected"
                 android:textStyle="bold" />
 
             <TextView
                 android:id="@+id/favorite_address"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
+                android:textSize="16sp"
```

Comment:
use dimen called "auto_fit_tv_text_size". Also you can rename that to "mybus_text_size" or something like that